### DISCUSSION TITLE
Sonoff Dual R3 - Power & Current

### DISCUSSION THREAD

**Discussion by adelafug** [arendst/Tasmota#14447] [Q&A] [answered]
Hi, I have a sonoff dual R3 with Tasmota 2022.01.2. how I can send power from second button to domoticz? 

In domoticz screen config only have "Sensor idx 4 Power,Energy" for first button, but not have nothing for second button.

Thanks

**Comment by barbudor** [comment, marked as answer]
While @joba-1 [message](https://github.com/arendst/Tasmota/issues/13815#issuecomment-1013652754) may be more pertinent (I don't own Domoticz) you can also find hwo to manually publish to Domotics in the Rules Cookbook such as this [one](https://tasmota.github.io/docs/Rules/#enable-or-disable-relay-with-a-switch-in-domoticz) or that other [one](https://tasmota.github.io/docs/Rules/#use-of-variables-and-tele-in-domoticz).

As you want to publish sensors, the 2nd one is probably the closest to what you want to achieve.
Of course trigger and domoticz ID have to be adapted....

**Reply by adelafug** [reply]
Ok, thanks. I review code of pluggin and Tasmota and test! Thanks!
